hack/deployer/runner/gke.go

```diff
@@ -143,6 +158,63 @@ func (d *GKEDriver) Execute() error {
 	return err
 }
 
+const (
+	GoogleComputeEngineStorageProvider = "pd.csi.storage.gke.io"
+)
+
+// setupLabelsForGCEProvider adds the "labels" parameter in the GCE storage classes.
+// These labels are automatically applied to GCE Persistent Disks provisioned using these storage classes.
+func (d *GKEDriver) setupLabelsForGCEProvider() error {
+	storageClassesYaml, err := exec.NewCommand("kubectl get sc -o yaml").WithoutStreaming().Output()
+	if err != nil {
+		return err
+	}
+	storageClasses := storagev1.StorageClassList{}
+	if err := yaml.Unmarshal([]byte(storageClassesYaml), &storageClasses); err != nil {
+		return err
+	}
+	labels, err := d.resourcesLabels()
+	if err != nil {
+		return err
+	}
+	for _, storageClass := range storageClasses.Items {
+		if storageClass.Provisioner != GoogleComputeEngineStorageProvider {
```

Comment:
Why exclude PV created by the `kubernetes.io/gce-pd` provisioner?

Reply:
> Why exclude PV created by the kubernetes.io/gce-pd provisioner?

Because it is the "in-tree provisioner" and it is supposed to be removed: https://kubernetes.io/blog/2021/12/10/storage-in-tree-to-csi-migration-status-update/#quick-recap-what-is-csi-migration-and-why-migrate
I think I can update the code to include it though.

Reply:
Alright, TIL. So I'm for leaving it that way.